### DISCUSSION TITLE
Latest release 9.4.0 few changes I notice that can give you small headache

### DISCUSSION THREAD

**Discussion by rapi3** [arendst/Tasmota#11859] [Q&A] [answered]
Hello,

_first:_ **Thank you all** for your work and new feature like: option for power consumption calculation, user settable antenna gain for MFRC522 RFID Reader etc.

_seccond:_ I just notice in latest release 9.4.0 few small undocumented changes:

1) **files run from UFs will run sa backlog0**, so no delay commands inside will be honored, previous version I had 9.3.1.x will run commands inside a file in UFS as backlog, not any more.
Before:
```
DisplayText [l1c1T][O]
Delay 25 ; now this line it is ignored
DisplayText [l1c1t]   [O]
```
Now I have to use this:
```
DisplayText [l1c1T][O]
Backlog Delay 25; Delay 25 ; now this line give me delay I need
DisplayText [l1c1t]   [O]
```
Not a major problem but need to be documented if will remain.

2) **Consoles menu**... this looks so confusing and hurt my eyes so I had to change for me to Mange... maybe we can make this permanent ? I can do a pull request for this small change if you agree.
![Screenshot_2021-04-23_21-08-33](https://user-images.githubusercontent.com/18531150/115918871-5c823500-a478-11eb-94e0-b7f6b8f89b03.png) ![Screenshot_2021-04-23_21-10-01](https://user-images.githubusercontent.com/18531150/115918900-66a43380-a478-11eb-8767-aadc6fcd5018.png)

3) **when publish %value% it capitalize all letters of original data value:** ( not sure if before this version it was ok )
I have simple rule to publish data received from RF & IR bridge:
```
ON RfReceived#Data do publish t/RF1/rfrec %value% ENDON
ON IrReceived#Data do publish t/RF1/irrec %value% ENDON
```

![Screenshot_2021-04-23_20-22-50](https://user-images.githubusercontent.com/18531150/115917912-21cbcd00-a477-11eb-95e7-f2331fb9b1ca.png)

This is important to know as can give you a headache until you will understand why you receive wrong data.

**Comment by sfromis** [comment, marked as answer]
Calling the Consoles menu "Manage" was rejected to avoid confusing people by an overly broad term, as a lot of the management happens in Configuration.... And of course "Firmware Upgrade" and "Restart" can as well be called management.

**Reply by rapi3** [reply]
For my it seems logic in Configuration need to be all I need to set for the device to work first and don't need to be changed after that all like are now but not without Timers.
In Manage I'd like to have the rest I use often: Console, Timers, UFS.
Firmware Upgrade for me it is ok to be under toggle button because few times already when I had there Configuration by mistake I pressed Toggle instead of Configuration and turned off a remote device.

**Reply by sfromis** [reply]
Whether you like it a certain does not make it any more logical to place a clear mangement feature outside a menu renamed to "Manage". Not saying that the current setup is "ideal", but IMO renaming to "Manage" makes it worse.

**Reply by rapi3** [reply]
Probably it is easy to move Manage File System menu on main interface or back to Configuration like before.

**Reply by sfromis** [reply]
Obviously, you can move things around in a local copy of yours, it *is* open source, including to be "like before", but that does not carry much weight if you think that it is "wrong" how it has been changed :-)

**Reply by Jason2866** [reply]
Any missing "stuff" noticed in the documentation is highly welcome to be changed.
The documentation is Open Source too. Just click the pencil and edit and submit a PR.
